flask/avogadro/src/server.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from flask import Flask, request
+
+import avogadro_api as avogadro
+
+app = Flask(__name__)
+
+
+@app.route('/calculate', methods=['POST'])
+def calculate():
+    json_data = request.get_json()
+    cjson = json_data['cjson']
```

Comment:
Lets add a little validation here. Checking the JSON contains the 'cjson' and 'mo' keys and return a status 400 if it doesn't.